src/custom-elements.js

```diff
@@ -182,17 +182,11 @@ let Deferred;
         throw new TypeError('constructor must be a Constructor');
       }
 
-      // 2. If constructor is an interface object whose corresponding interface
-      //    either is HTMLElement or has HTMLElement in its set of inherited
-      //    interfaces, throw a TypeError and abort these steps.
-      //
-      // It doesn't appear possible to check this condition from script
-
-      // 3:
+      // 2:
       const nameError = checkValidCustomElementName(name);
       if (nameError) throw nameError;
 
-      // 4, 5:
+      // 3:
```

Comment:
The numbering has changed in the spec for `customElements.define`, so I updated it.